syntax = "proto2";

package shardora.timeblock.protobuf;

message TimeBlock {
    optional uint64 height = 1;
    optional uint64 timestamp = 2;
    optional uint64 vss_random = 3;
}
